### PR TITLE
Added writing a "develop" version, using the lowest (last) entry.

### PR DESCRIPTION
**Pull request by awholegunch**
Also, updates for 4.2.0-SNAPSHOT.

### CODE REVIEW

File: docs.cask.co/scripts/builder.py [hunk 1]
```diff
@@ -17,11 +17,15 @@
...
 
-# version 0.5
+# version 0.6
 
 # JSON Creation (json-versions.js)
 # Creates a JSON file with timeline and formatting information from the data in a 
 # supplied configuration file.
+#
+# Writes out the current version to a file "VERSION" and 
+# the last development version to a file "DEVELOPMENT"
```
Comment (wolf31o2) [inline docs.cask.co/scripts/builder.py]: The files are `version` and `develop`

File: docs.cask.co/scripts/builder.py [hunk 2]
```diff
@@ -323,16 +341,22 @@ def converted_line(line):
...
 def write_js_versions(target):
+    files = []
     target_dir = os.path.dirname(target)
-    version = os.path.join(target_dir, 'version')
+    if get_json_versions():
+        files.append((target, get_json_versions()))
+    if get_current_version():
+        files.append((os.path.join(target_dir, 'version'), get_current_version()))
+    if get_develop_version():
+        files.append((os.path.join(target_dir, 'develop'), get_develop_version()))
```
Comment (wolf31o2) [inline docs.cask.co/scripts/builder.py]: Set `version` and `develop` as variables, then check/assign using those variables, so you don't run the parsing functions twice.

**Review by wolf31o2 — CHANGES_REQUESTED** [review]

File: docs.cask.co/scripts/builder.py (hunk 1 above)
Comment (awholegunch) [reply docs.cask.co/scripts/builder.py]: Fixed.

File: docs.cask.co/scripts/builder.py (hunk 2 above)
Comment (awholegunch) [reply docs.cask.co/scripts/builder.py]: Fixed.

**Review by wolf31o2 — APPROVED** [review]

**Pull request merged**